config/values/earlyyears.yml

```diff
@@ -1,4 +1,3 @@
-## implemented above this line
 earlyyears:
   bursarywithfirstclassdegree: £5,000
```

Comment:
@Sarah-DfE We are a bit inconsistent with the variable names for bursaries:

```
bursarywithfirstclassdegree: £5,000
bursarywithtwoone: £4,000
bursarywithtwotwo: £2,000
```

Should we change to either:
```
bursarywithfirstclassdegree: £5,000
bursarywithtwoonedegree: £4,000
bursarywithtwotwodegree: £2,000
```

or 

```
bursarywithfirst: £5,000
bursarywithtwoone: £4,000
bursarywithtwotwo: £2,000
```

or something else?

Reply:
Agreed with @Sarah-DfE that we should simplify and make more consistent as:

```
bursarywithfirst: £5,000
bursarywithtwoone: £4,000
bursarywithtwotwo: £2,000
```

Reply:
@gemmadallmandfe I can't see that the undergraduate training cost of £9,250 is included above, it's on the preview page